types/evidence.go

```diff
@@ -367,6 +403,71 @@ func (l *LightClientAttackEvidence) ValidateBasic() error {
 	return nil
 }
 
+// ValidateABCI
```

Comment:
This doesn't tell us anything ;-)

Reply:
Lol good catch - I must've got lazy the second time I had to write it